doc/whats-new.rst

```diff
@@ -20,6 +20,7 @@ v0.10.1 (unreleased)
 
 Enhancements
 ~~~~~~~~~~~~
+- Added nodatavals attribute to dataset when using open_rasterio.
```

Comment:
If you want you can credit yourself here. If you do so it would be good also to link to the original issue (see examples below) and to rename dataset do `DataArray`